src/coredump/coredump.c

```diff
@@ -1212,8 +1223,23 @@ static int process_kernel(int argc, char* argv[]) {
 
         if (context.is_journald || context.is_pid1)
                 r = submit_coredump(&context, iovw, STDIN_FILENO);
-        else
+        else {
+                process_name  = filename_escape(context.meta[META_COMM]);
+                if (!process_name) {
+                        log_error_errno(r, "Failed to escape process name. Ignoring coredump ratelimit %m");
+                } else {
+                        /* per process coredump ratelimit */
+                        r = coredump_ratelimit(process_name, arg_ratelimitinterval, arg_ratelimitburst);
+                        if(r < 0) {
+                                log_error_errno(r, "Failed to process coredump ratelimit %m");
```

Comment:
we usually would put a `: ` before the `%m`, to separate it from the sentence.

Reply:
Addressed the comment. Please check once.